soc/nxp/imxrt/Kconfig.soc

```
@@ -0,0 +1,10 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_IMXRT
+	bool
+
+config SOC_FAMILY
+	default "imxrt" if SOC_FAMILY_IMXRT
```

Same issue as other pr:
```suggestion
config SOC_FAMILY_NXP_IMXRT
	bool

config SOC_FAMILY
	default "nxp_imxrt" if SOC_FAMILY_NXP_IMXRT
```